src/config/config.py

```diff
@@ -14,8 +14,9 @@
     'autoreload': True,
     'template_path':'templates/',
     'static_path':'static/',
+    'login_url': '/login'
```

Comment:
That is not a route! See [this](http://guillaumevincent.com/2013/02/12/Basic-authentication-on-Tornado-with-a-decorator.html). That is a configuration option that tells Tornado sitewide login is at `/login`